charts/mlp/values.yaml

```diff
@@ -0,0 +1,118 @@
+deployment:
+  image:
+    pullPolicy: IfNotPresent
+    registry: ghcr.io
+    repository: gojek/mlp
+    tag: v1.7.1
+  replicaCount: 1
+
+  # Additional labels to apply on the pod level
+  podLabels: {}
+
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources: {}
+  #   requests:
+  #     cpu: 100m
+  #     memory: 128Mi
+
+  livenessProbe:
+    path: "/v1/internal/live"
+  readinessProbe:
+    path: "/v1/internal/ready"
+
+  apiHost: "http://mlp/v1"
+  mlflowTrackingUrl: "http://mlflow.mlp"
+  environment: "production"
+
+  authorization:
+    enabled: false
+    serverUrl: http://mlp-authorization-keto
+
+  teams:
+    - marketing
+    - operation
+    - business
+  streams:
+    - promotion-marketing
+    - operation-strategy
+    - business-analyst
+  docs:
+    [
+      {
+        "label": "Merlin User Guide",
+        "href": "https://github.com/gojek/merlin/blob/main/docs/getting-started/README.md",
+      },
+      {
+        "label": "Turing User Guide",
+        "href": "https://github.com/gojek/turing",
+      },
+      {
+        "label": "Feast User Guide",
+        "href": "https://docs.feast.dev/user-guide/overview",
+      },
+    ]
+  ui:
+    feastCoreApi: "http://feast.dev/v1"
+    merlinApi: "http://merlin.dev/v1"
+    turingApi: "http://turing.dev/v1"
+    clockworkHomepage: "http://clockwork.dev"
+    feastHomepage: "http://feast.dev"
+    kubeflowHomepage: "http://kubeflow.org"
+    merlinHomepage: "http://merlin.dev"
+    turingHomepage: "http://turing.dev"
+
+ingress:
+  enabled: false
+  # class:
+  # host:
+  # path: "/"
+
+encryption:
+  # encryption.key must be specified using --set flag.
+  key: "example-key-here"
+
+service:
+  externalPort: 8080
+  internalPort: 8080
+
+# If you would like to use an external postgres database, you can connect using these credentials
+externalPostgresql:
+  enabled: false
+  username: mlp
+  database: mlp
+  password: password
+  address: 127.0.0.1
+
+postgresql:
+  enabled: true
+  ## Configure resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ##
+  resources: {}
+  #   requests:
+  #     cpu: 100m
+  #     memory: 128Mi
+  persistence:
+    size: 10Gi
+  nameOverride: mlp-postgresql
```

Comment:
why do we override the name here?

Reply:
When postgres is enabled explicitly for mlp (under mlp values in caraml values)
* postgres will be created with name caraml-mlp-postgresql (<release_name>-<chart_name>-postgresql)
* user and db are created as mlp reading from the values under mlp chart
* mlp will use the values from its values file

By default, Postgresql will create the pods with name <release_name>-postgresql. To add the chart name in the pods that are created, we need to override the name. So that the created pods/service/deployment will have the name `caraml-mlp-postgresql`

Reply:
but it will be a conflict if for some reason we install two CaraML releases

Reply:
the release name would be different if there are two CaraML releases, right? The prefix there will change. 

If I have two releases with names `caraml`, `caraml-v2` then the databases would be `caraml-mlp-postgresql` and `caraml-v2-mlp-postgresql`

Reply:
oh, that's cool!